views/jira-manual-app-creation.hbs

```diff
@@ -0,0 +1,142 @@
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="ap-local-base-url" content="{{localBaseUrl}}" />
+    <meta name="public-url" content="{{APP_URL}}" />
+    <title>{{title}}</title>
+    <link
+      rel="stylesheet"
+      href="/public/aui/aui-prototyping.css"
+      integrity="DTM1Q+8lU7SzJT+FWr0JFisCSZlwfM0GiAKYy7h1s9vIKa/CIh37s9NuOCqIOgK4tmqrjLK4NuWuIPUQNsikHA=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="stylesheet" href="/public/css-reset/bundle.css" media="all" />
+    <link
+      rel="stylesheet"
+      href="/public/atlassian-ui-kit/bundle.css"
+      media="all"
+    />
+    <link rel="stylesheet" href="/public/css/global.css" media="all" />
+    <link
+      rel="stylesheet"
+      href="/public/css/jira-manual-app-creation.css"
+      media="all"
+    />
+    <script src="/public/js/jquery.min.js" nonce="{{nonce}}"></script>
+  </head>
+
+  <body class="aui-page-hybrid">
+  {{> navigation previousPageName="Jira Select App Creation" }}
+
+  <div class="jiraManualAppCreation">
+
+      <h1>Manual app Creation</h1>
+
+      <div>Learn how to <a href="#">manually set up a GitHub app</a>. All fields are required.</div>
+
+      <form id="jiraManualAppCreation__form" class="aui top-label" enctype="multipart/form-data">
+        <div class="jiraManualAppCreation__formTitle">
+          Details
+          <i class="aui-icon aui-iconfont-question-filled"></i>
+        </div>
+
+        <div class="field-group">
+          <label for="app-name">GitHub app name</label>
+          <input class="text full-width-field" type="text" required id="app-name" name="app-name" />
+        </div>
+
+        <div class="field-group ">
+          <label for="app-id">GitHub app ID</label>
+          <input class="text full-width-field" type="text" required id="app-id" name="app-id" />
+        </div>
+
+        <div class="field-group">
+          <label for="client-id">GitHub client ID</label>
+          <input class="text full-width-field" type="text" required id="client-id" name="client-id" />
+        </div>
+
+        <div class="jiraManualAppCreation__formTitle">URLs</div>
+        <div class="jiraManualAppCreation__formSubTitle">
+          Copy and paste the following URLs into your GitHub Enterprise Server configuration. This is unique to your site.
+        </div>
+
+        <div class="field-group">
+          <label for="callback-url">Callback URL</label>
+          <!--TODO: replace the value by actual data-->
+          <input class="text full-width-field" type="text" id="callback-url"
+                 name="callback-url" value="https://github.atlassian.com/github/callback/V1StGXR8_Z5jdHi6B-myT"
```

Comment:
These IDs need to be programmatically generated. This is another thing you can choose to tackle as part of this PR or separately.

Reply:
I thought this would be generated and supplied from the backend. If we do it in the frontend, how do we generate the link?